internal/store/pod.go

```diff
@@ -375,6 +376,32 @@ func createPodContainerStatusLastTerminatedExitCodeFamilyGenerator() generator.F
 	)
 }
 
+func createPodContainerStatusLastTerminatedTimestampFamilyGenerator() generator.FamilyGenerator {
+	return *generator.NewFamilyGeneratorWithStability(
+		"kube_pod_container_status_last_terminated_timestamp",
+		"Last terminated time for a pod container, when container was terminated with kube_pod_container_status_last_terminated_reason, in unix timestamp.",
```

Comment:
```suggestion
		"Last terminated time for a pod container in unix timestamp.",
```
Is `when container was terminated with kube_pod_container_status_last_terminated_reason` necessary here?

Reply:
Hey @rexagod , thank you for the feedback! I've tried to be more precise in the description, but I think you are right, there is no need to keep it.
Cleaned it up in this commit - https://github.com/kubernetes/kube-state-metrics/pull/2291/commits/393430ed86941c502e179deffff625fee1864f3b, please have a look